code/modules/hallucinations/effects/moderate.dm

```diff
@@ -284,9 +284,9 @@
 		var/turf/source_turf = t
 		expand_queue -= source_turf
 		// Expand to each dir
-		for(var/dir in GLOB.cardinal)
-			var/turf/target_turf = get_step(source_turf, dir)
-			if(processed[target_turf] || !source_turf.CanAtmosPass(target_turf))
+		for(var/direction in GLOB.cardinal)
```

Comment:
You don't need to change this but this is actually the worst way to do this vs a heap. We need to get rid of GLOB.cardinal and make into a macro

Reply:
I've updated the ones that I significantly changed to use RANGE_TURFS and/or RANGE_EDGE_TURFS. We should probably do a cleanup CL to get the rest at some point.